### PR TITLE
Migrate from SemanticUI to MUI

### PR DESCRIPTION
**Pull request by adrianbona**

### CODE REVIEW

File: src/style/theme.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        },
+        MuiRadio: {
+            size: 'small',
+        },
+        MuiSwitch: {
+            size: 'small',
+        },
+        MuiTextField: {
+            margin: 'dense',
+            size: 'small',
```
Comment (ACatThatPrograms) [inline src/style/theme.js:102]: Just mu curiosity, are these prop defaults?

**Review by ACatThatPrograms — APPROVED** [review]
Some comments, but I think we are 100%

Excellent work, truly I don't have enough good things to say on this giant task.

File: package.json [hunk 2]
```diff
@@ -3,13 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/free-brands-svg-icons": "^6.2.0",
+    "@fortawesome/free-regular-svg-icons": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply package.json:12]: Oh no. Semantic was using FA. And MUI doesn't have icons for brands like the ones we use for Windows, Linux and iOS. I decided to add those back.

File: src/components/header/Header.jsx [hunk 3]
```diff
@@ -1,40 +1,198 @@
...
+                        window.open(WALLET_WINDOWS_URL, '_blank').focus();
+                        handleClose();
+                    }}
+                >
+                    <FontAwesomeIcon icon={faWindows} />
+                    Windows
+                </MenuItem>
+            </Menu>
+        </div>
+    );
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply src/components/header/Header.jsx:98]: Yup, the problem was what MUI Icons were lacking.

File: src/components/statusOverlay/StatusOverlay.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    DialogTitle,
+    Typography
+} from "@mui/material";
+
+export const StatusOverlay = () => {
+
+    useSelector(s => s);
+
+    const busy = aliceNetAdapter.busy || ""; // Todo: Update to support all required busy states
+    const error = aliceNetAdapter.error || ""; // Todo: Update to support all required errors
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply src/components/statusOverlay/StatusOverlay.jsx:22]: Those were left untouched!

File: src/pages/blockExplorer/BlockExplorer.jsx [hunk 5]
```diff
@@ -40,15 +47,7 @@ export function BlockExplorer() {
...
-        return (
-            <Page>
-                <Grid>
-                    <Dimmer active>
-                        <Loader>Loading</Loader>
-                    </Dimmer>
-                </Grid>
-            </Page>
-        );
+        return null;
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply src/pages/blockExplorer/BlockExplorer.jsx:50]: I've simplified the loading and the error display by using a single component (StatusOverlay). That component is being globally used when loading any content, therefore I've removed any "loading" renders for the rest of the pages.

File: src/pages/dataExplorer/DataExplorer.jsx [hunk 6]
```diff
@@ -26,17 +25,8 @@ export function DataExplorer() {
...
-        return (
-            <Page>
-                <Grid>
-                    <Dimmer active>
-                        <Loader>Loading</Loader>
-                    </Dimmer>
-                </Grid>
-            </Page>
-        );
+        return null;
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply src/pages/dataExplorer/DataExplorer.jsx:29]: Answered for BlockExplorer.jsx

File: src/pages/txExplorer/TxExplorer.jsx [hunk 7]
```diff
@@ -39,15 +39,7 @@ export function TxExplorer() {
...
-        return (
-            <Page>
-                <Grid>
-                    <Dimmer active>
-                        <Loader>Loading</Loader>
-                    </Dimmer>
-                </Grid>
-            </Page>
-        );
+        return null;
```
[the remark this answers is not in the document]
Comment (adrianbona) [reply src/pages/txExplorer/TxExplorer.jsx:42]: Answered for BlockExplorer.jsx

File: src/style/theme.js (hunk 1 above)
Comment (adrianbona) [reply src/style/theme.js]: Those can be removed. Thanks for asking!

File: src/pages/blockExplorer/BlockExplorer.jsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (ACatThatPrograms) [reply src/pages/blockExplorer/BlockExplorer.jsx:50]: Perfect!

**Comment by ACatThatPrograms** [comment]
:100: Nicely done

**Pull request merged**